mega-zep-frontend/src/main/angular/frontend/src/app/modules/shared/constants/configuration.ts

```diff
@@ -8,7 +8,6 @@ export const configuration = {
   },
 
   /* TODO: Remove these obsolete properties, because these properties are provided via the /config endpoint */
-  SPREADSHEET_URL: 'https://docs.google.com/spreadsheets/d/1wUJHMtkY47RhLIGytg_MXVe_hgPAj_yzozLMcv5hrdU/',
   ZEP_URL: 'https://www.zep-online.de/zepgepardecservices',
```

Comment:
@Ruhsi  This url should also be provided by the backend config endpoint. If possible, then add it